libindy/src/services/wallet/mod.rs

```diff
@@ -1576,4 +1686,169 @@ mod tests {
         assert_eq!("type", record.get_type().unwrap());
         assert_eq!("value1", record.get_value().unwrap());
     }
-}
+
+    /**
+        Export/Import tests
+    */
+    fn _get_export_dir_path() -> &'static str {
+         "/tmp/indy_wallet_export_tests"
```

Comment:
Will it work on windows and OSX? There is corresponded You can use EnvironmentUtils::tmp_path